examples/js/shared.js

```diff
@@ -49,6 +49,24 @@ let serverSecret = null
 const app = express()
 app.use(express.json())
 
+const checkEnvVars = () => {
+    if (!authUrl) {
+        throw new Error('AUTH_URL is undefined!')
+    }
+    if (!apiUrl) {
+        throw new Error('API_URL is undefined!')
+    }
+    if (!apiKey) {
+        throw new Error('API_KEY is undefined!')
+    }
+    if (!clientId) {
+        throw new Error('CLIENT_ID is undefined!')
+    }
+    if (!clientSecret) {
+        throw new Error('CLIENT_SECRET is undefined!')
+    }
+}
```

Comment:
Since api-lifecycle test requires user id and org id to be set, I think you should check them here.